sql-aggregates/starring-in.sql

```diff
@@ -0,0 +1,8 @@
+select "g"."name" as "genre",
+       count(*) as "numberOfGenre"
+  from "actors"
+  join "castMembers" using ("actorId")
+  join "filmGenre" using ("filmId")
+  join "genres" as "g" using ("genreId")
+ where "actorId" = 178
```

Comment:
I guess you looked up Lisa Monroe to find the actorId. You could also query against `firstName = 'Lisa' and lastName = 'Monroe'.